lorem.html

```diff
@@ -49,17 +53,18 @@
         </nav>
       </header>
       <main>
-        <h1>
+        <h1>Lorem Ipsum</h1>
```

Comment:
The only thing that should be nested inside your headings is the heading text.  Here you nested the whole page contents inside the heading element